README.md

```diff
@@ -1,41 +1,57 @@
-# zosia_print
+# Zosia-print
 
-Zosia print is a project that aims to automate creation of all printables that are needed at ZOSIA conference.
+Zosia-print is a project that aims to automate creation of all printables that are needed at the ZOSIA conference. Based on HTML templates PDF files are generated using [Weasyprint](https://github.com/Kozea/WeasyPrint).
+
+This repository also [contains information](/data/) regarding lectures schedules over years and places description.
 
 ## Overview of solution
 
-- We generate printables using following data:
-  * places.yaml - contains descriptions of attractions that are availble nearby, needs to be updated manually to fit the location/
-  * schedule.yaml - contains schedule of the conference. Here is a quick overview of the format:
-    ~~~
-    - name: <Name on the plan>
-      session_name: <Name near the lecture description>
-      events:
-              - title: <event title> (must match title on the website if it's a lecture)
-                type: <event type> (possible values: LECTURE, MEAL)
-                startTime: <start time> (using format "hh:mm")
-                duration: <length of event in minutes>
-    ~~~
-  * data.json - json file to be exported from zosia website
-- We use gen.py script to generate the .html files from jinja templates and files mentioned above.
-- Generated .html files are transformed into pdfs using a tool called weasyprint.
+- We generate HTML templates for Weasyprint using following data:
+  * places - yaml file that contains descriptions of attractions that are available nearby, needs to be updated manually to fit the location. Those files are kept in this repository.
+  * schedule - csv file that contains schedule of the conference generated in the scheduler-spreadsheet.
```

Comment:
Are `places` and `schedules` files or directories? If the latter, it should be properly stated, e.g. `places - directory with YAML files`.